sa/_db-next/migrations/20171006131843_FinalizeOrder.sql

```diff
@@ -0,0 +1,20 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE orders DROP COLUMN csr;
+
+CREATE TABLE requestedNames (
+  `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+  `orderID` BIGINT(20) NOT NULL,
+  `reversedName` varchar(640) CHARACTER SET ascii NOT NULL,
```

Comment:
@rolandshoemaker I matched this to the `CREATE TABLE` syntax for `issuedNames`:
>   `reversedName` varchar(640) CHARACTER SET ascii NOT NULL,

I'm open to changing it but thought there might have been a specific reason `issuedNames` was specified this way.

Reply:
Ah, I suspect the original reason for this was to prevent any non-punycode encoded unicode names ending up in this table, which is a bit silly because if this is our last line of defense something else has broken pretty hard somewhere else...

That said I'm fine keeping it because I can't come up with a good reason to get rid of it ¯\_(ツ)_/¯.

Length wise unless there is a good reason I think we should reduce it, mainly because we'll never actually use all of the width and it then matches the belt and bracers approach of setting the character set...